easybuild/easyblocks/b/boost.py

```diff
@@ -89,6 +89,8 @@ def patch_step(self):
     def configure_step(self):
         """Configure Boost build using custom tools"""
 
+        configopts = "--prefix=%s" % self.objdir
```

Comment:
@wpoely86 is right, don't worry about list values in the easyconfig file for `configopts`, the `configure_step` will run for each of the values once

@torbjoernk: my syntax was wrong, use `self.cfg.update('configopts', "...")`